### ISSUE TITLE
Training Model issue

### ISSUE THREAD

**Issue by yzzupgo** [FSE2021anonymous/Recoder#16]
Hello! Thanks for open-sourcing the work!
I am trying to train the model without the docker. However, I encounter the error shown in the figure.
![6ceaf2624b4e6793be0bf250bffc88f](https://user-images.githubusercontent.com/98015651/210310667-0dc66a8c-6645-4a7e-afc4-ea703502ce7c.png)
The error is caused by line 151 of run.py. Specifically, the variable test_set is defined on line 150 of run.py, and the argument "test" is passed to SumDataset to initialize test_set. However, the third branch of the __init__ function in the Dataset.py does not do anything, which causes self.data to be empty.
Furthermore, could you provide the files of the test set and validation set (i.e., valdata.pkl, valnl.pkl, testdata.pkl, testnl.pkl, testcopy.pkl).
Thank you for taking a look.

**Comment by pkuzqh** [comment]
you can copy them from the docker

**Comment by yzzupgo** [comment]
Thanks for your quick reply.
